src/main/resources/style/taskcardstyles/TaskCardBaseStyle.css

```diff
@@ -0,0 +1,59 @@
+#taskCard {
```

Comment:
I don't know how JavaFX CSS works, but in general it's very bad practice to use `id`s for repeating elements in HTML.